modules/ROOT/pages/amazon/amazon-sns-connector-reference.adoc

```diff
@@ -1304,46 +1301,121 @@ The Unsubscribe action deletes a subscription. If the subscription requires auth
 * SNS:RETRY_EXHAUSTED
 * SNS:PLATFORM_APPLICATION_DISABLED
 
+
+
 == Types
+[[Tls]]
+=== Tls
```

Comment:
Change "=== Tls" to "=== TLS" -- all caps is correct

Reply:
Isn't this in the code? It is a type.

Reply:
Probably, but we can make it more readable and users can easily understand the choice in Studio.

Reply:
But I think users have to write it as Tls because that is what the code accepts.